autorag/nodes/passagereranker/sentence_transformer.py

```diff
@@ -36,22 +35,13 @@ def sentence_transformer_reranker(queries: List[str], contents_list: List[List[s
     nested_list = [list(map(lambda x: [query, x], content_list)) for query, content_list in zip(queries, contents_list)]
     rerank_scores = flatten_apply(sentence_transformer_run_model, nested_list, model=model, batch_size=batch)
 
-    df = pd.DataFrame({
-        'contents': contents_list,
-        'ids': ids_list,
-        'scores': rerank_scores,
-    })
-
-    df[['contents', 'ids', 'scores']] = df.apply(sort_by_scores, axis=1, result_type='expand')
-    df['contents'] = df['contents'].apply(lambda x: x[:top_k])
-    df['ids'] = df['ids'].apply(lambda x: x[:top_k])
-    df['scores'] = df['scores'].apply(lambda x: x[:top_k])
+    sorted_contents, sorted_ids, sorted_scores = select_top_k(contents_list, ids_list, rerank_scores, top_k)
```

Comment:
`select_top_k` has only two parameters and your input is four...?
You did not even test this code.
Please **TEST YOUR CODE** before requesting review for merge. 

![image](https://github.com/Marker-Inc-Korea/AutoRAG/assets/28955029/4bf56aaf-b011-4f94-a503-3dd58c21ced2)